index.js

```diff
@@ -1449,11 +1449,11 @@
     impl: curry2 (Z.contramap)
   };
 
-  //. ### Combinator
+  //. ### Combinator: A function or definition with no free variables.
```

Comment:
I don't know anything about Church's thesis. :stuck_out_tongue_winking_eye:

I'd like to make a comment about this change which does not relate to your comment above. The following links are generated from the heading text:

  - <https://github.com/sanctuary-js/sanctuary#combinator>
  - <https://sanctuary.js.org/#combinator>

I'd like to preserve these elegant URLs. We could make your proposed addition the section's opening paragraph rather than part of its heading.